### DISCUSSION TITLE
Feature request: Use a sound file or text to speech (TTS) in a new "AUDIO" output configs tab

### DISCUSSION THREAD

**Discussion by vinKaiZen** [MobiFlight/MobiFlight-Connector#1681] [Ideas]
**Is your feature request related to a problem? Please describe.**
It should be possible to use a sound file or text to speech (TTS) in the output configs if a config line condition is fullfilled.
(Example usage for checklist items, callouts..)

**Describe the solution you'd like**
In the output config area, there should be a new tab "AUDIO" (example behind the display tab) for a config line.
There you could choose an output audio device, a soundfile (mp3, aac or similar)
A second choose could be a text field for text to speech (TTS)

**Comment by DocMoebiuz** [comment]
Please discuss this here in discussions first and see how this resonates with other users
